docs/en/observability/cloud-monitoring/aws/monitor-amazon-ec2.asciidoc

```diff
@@ -0,0 +1,372 @@
+[[monitor-amazon-ec2]]
+= Monitor Amazon Cloud Compute (EC2)
+
+++++
+<titleabbrev>Monitor EC2</titleabbrev>
+++++
+
+Amazon Elastic Compute Cloud (Amazon EC2) enables you to provision compute
```

Comment:
Add generic link to info about the service. Make sure the link is "future proof" (unlikely to change): https://docs.aws.amazon.com/ec2/index.html